README.md

```diff
@@ -145,3 +145,17 @@ use ApplicationInsights::Rack::TrackRequest, '<YOUR INSTRUMENTATION KEY GOES HER
 # For rails, suggest to set up this middleware in application.rb so that unhandled exceptions from controllers are also collected
 config.middleware.use 'ApplicationInsights::Rack::TrackRequest', '<YOUR INSTRUMENTATION KEY GOES HERE>', <buffer size>
 ```
+
+#### Retrieving a request's AppInsight ID  ####

```

Comment:
I think you mean the RequestId of the request?
AppInsights ID feels like the id of the AppInsights resource is Azure, something like instrumentation key.